### PR TITLE
Create __main__.py

### LINKED ISSUE #1: Add __main__.py
It would be useful to have a \_\_main\_\_.py file to allow this to run as a command-line application.

### PR DESCRIPTION
**Pull request by Beakerboy**
- Resolve #1

### CODE REVIEW

**Comment by kshitijrajsharma** [comment]
Thanks for the PR , I will request to add documentation for this so that people can run using documentation

**Comment by kshitijrajsharma** [comment]
Thanks for the changes , Please give me this week to review

File: README.md [hunk 1]
```diff
@@ -13,16 +13,5 @@ pip install geojson2osm
 ## Usage
 
 ```python
-import json
```
Comment (kshitijrajsharma) [inline README.md:16]: Lets include both , the import  as a python lib and from the command line

**Review by kshitijrajsharma — COMMENTED** [review]
Thank you for this Awesome PR , I have added one comment : Once resolved good to merge

File: README.md (hunk 1 above)
Comment (Beakerboy) [reply README.md:16]: Done.

**Review by kshitijrajsharma — APPROVED** [review]

**Pull request merged**